tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
-    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing success": "${{name}} estias kaptita!",
     "combat_capturing_fail": "${{name}} liberiĝis!",
     "combat_replacement": "Elektu anstataŭanton!",
     "combat_replacement_is_fainted": "La monstro svenis!",
```

Comment:
I'm not sure about "jam", as it is translated as "already". The translation would be "The monster is already fainted". It seems that it battled a little bit then it fainted as not so powerful. 
I'm not sure about "Tiu" instead of "La", too: "That monster is fainted". "That" who? 
Instead I suggest "Via" instead of "La": "Via monstro svenis", "Your monster is fainted". 
If you want to change the entire key you should see also the [original one](Tuxemon/Tuxemon/blob/development/tuxemon/resources/db/locale/en_US.json#L15) ;)